### DISCUSSION TITLE
Consistently display arrow icon in combobox

### DISCUSSION THREAD

**Discussion by stumcrew** [twilio-labs/paste#1033] [Q&A] [answered]
Hi all. We've recently updated some dropdowns in Flex settings in Console to the new combobox component. We're using two different types of combobox - basic (which shows the arrow icon) and autocomplete (which doesn't). The team decided to add the search icon to show users what to do in the comboboxes with no icon. A better solution would be to show the dropdown arrow in all of these combo boxes. Is there a way to add the dropdown arrow icon into the boxes that don't show it by default? Thanks!
![Screenshot 2020-12-11 at 11 29 55](https://user-images.githubusercontent.com/59792516/101920606-ee715b80-3bc3-11eb-91dd-df4ed13fa9df.png)

**Comment by TheSisb** [comment]
A couple options:

- You can use `insertBefore` for the search icon,  so the chevron down icon remains on the right.
- You can add both the search and the chevron down icon (`import {ChevronDownIcon} from '@twilio-paste/icons/esm/ChevronDownIcon';`) in the `insertAfter`.

**Reply by stumcrew** [reply]
If we use `insertAfter` to add the `ChevronDownIcon` it will display in a grey box, which will look different to the other two. Can the background of the `insertAfter` be removed?

**Reply by TheSisb** [reply]
Not at this time unfortunately. I think putting the search icon in the insertBefore may be the best path for now.

**Comment by aayushpi** [comment, marked as answer]
Hi @stumcrew I would also recommend opportunistically upgrading the forms in taskrouter to use the new label styles. They are available in the combobox component as the `labelText` prop
